### PR TITLE
Changes all instances of sulfuric acid to sulphuric acid

### PR DESCRIPTION
**Pull request by Simplehorror**
# Document the changes in your pull request

Consistency reasons, most places should be using the european english version.
Closes #16913

# Spriting

# Wiki Documentation

# Changelog

:cl:  
spellcheck: Changes all instances of sulfuric acid to sulphuric acid
/:cl:

### CODE REVIEW

**Comment by Maia-J** [comment]
> all instances of sulfuric acid to sulphuric acid

Changes exactly 0 of said instances and instead changes instances of Sulfur

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -850,8 +850,8 @@
 	M.adjustOrganLoss(ORGAN_SLOT_BRAIN, 1)
 	..()
 
-/datum/reagent/sulfur
-	name = "Sulfur"
+/datum/reagent/sulphur
+	name = "sulphur"
```
Comment (tattax) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm:854]: 😭

**Comment by missatessatessy** [comment]
Seems like it would be more convenient to just change instances of sulphur to sulfur #16913 . Since there are only 5 "Sulphur" in the code, and majority of coders and userbase are American.

**Comment by Simplehorror** [comment]
> Seems like it would be more convenient to just change instances of sulphur to sulfur #16913 . Since there are only 5 "Sulphur" in the code, and majority of coders and userbase are American.

ok colonizer, to be honest I don't think anything needs changing since this is a multicultural codebase

**Comment by neeshacark** [comment]
we have been changing stuff to the american variant for literal years, almost everything is american for the love of god dont make this an outlier our poor wiki staff

**Comment by ToasterBiome** [comment]
USA! USA! USA!

**Review by MajManatee — APPROVED** [review]
Rule Britania

**Review by JamieD1 — APPROVED** [review]

**Comment by ToasterBiome** [comment]
I will not stand for this.

**Pull request merged**